### DISCUSSION TITLE
[Question]Is the cleaning of Docker cache the same as native Docker? And how to implement detailed header transmission rules like Nginx ingress?

### DISCUSSION THREAD

**Discussion by Ophira0110** [kubevela/kubevela#909] [Q&A]

**Comment by ryanzhang-oss** [comment]
This is not really related to Kubevela

**Comment by hongchaodeng** [comment]
Hi @Ophira0110 

Q: Is the cleaning of Docker cache the same as native Docker? 
A: Yes. Vela currently uses local `docker` command for `build` step. 

Q: How to implement detailed header transmission rules like Nginx ingress?
A: Under the hood, Vela encapsulates Ingress resource via CUE template. To implement it needs changes to CUE template in related definition which includes adding parameters and ingress config.
